.eslintignore

```diff
@@ -0,0 +1,17 @@
+# Files and folders related to build/test tools
+/build
+/node_modules
+/tests
+/vendor
+/tools
+/Gruntfile.js
+/webpack.config.js
+
+# Excluded files and folders based on `jsdoc.conf.json` exclusions
+/src/js/_enqueue/vendor
```

Comment:
The following files in this PR have changes, though they are ignore due to this ESLint exclusion, were these files change intentionally here in this PR @whyisjake?

```
src/js/_enqueues/vendor/codemirror/csslint.js
src/js/_enqueues/vendor/mediaelement/wp-mediaelement.js
src/js/_enqueues/vendor/plupload/moxie.js
src/js/_enqueues/vendor/plupload/plupload.js
src/js/_enqueues/vendor/plupload/wp-plupload.js
src/js/_enqueues/vendor/tinymce/plugins/wptextpattern/plugin.js
src/js/_enqueues/vendor/tinymce/tiny_mce_popup.js
```

Reply:
This might have been done ahead of adding them to the ignore.

Reply:
Updated in the commit. 

I had `_enqueue` and not `_enqueues` in the ignore file.